x-pack/plugins/secops/public/components/timeline/body/renderers/auditd_was_authorized_details.tsx

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { EuiSpacer } from '@elastic/eui';
+import { get } from 'lodash/fp';
+import * as React from 'react';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+
+import { BrowserFields } from '../../../../containers/source';
+import { Ecs } from '../../../../graphql/types';
+import { DraggableBadge } from '../../../draggables';
+
+import { PrimarySecondaryUserInfo } from './primary_secondary_user_info';
+import { SourceDest } from './source_dest_ip';
+
+import * as i18n from './translations';
+
+const Details = styled.div`
+  margin: 10px 0px 10px 10px;
+`;
+
+const TokensFlexItem = styled(EuiFlexItem)`
+  margin-left: 3px;
+`;
+
+interface Props {
+  id: string;
+  hostName: string | null | undefined;
+  userName: string | null | undefined;
+  primary: string | null | undefined;
+  secondary: string | null | undefined;
+  processExecutable: string | null | undefined;
+  processTitle: string | null | undefined;
+  workingDirectory: string | null | undefined;
+  args: string | null | undefined;
+  session: string | null | undefined;
+}
+
+export const AuditdWasAuthorizedLine = pure<Props>(
+  ({
+    id,
+    hostName,
+    userName,
+    primary,
+    secondary,
+    processExecutable,
+    processTitle,
+    workingDirectory,
+    args,
+    session,
+  }) => (
+    <EuiFlexGroup justifyContent="center" gutterSize="none" wrap={true}>
+      <TokensFlexItem grow={false}>{i18n.SESSION}</TokensFlexItem>
+      <TokensFlexItem grow={false}>
+        <DraggableBadge
+          id={`auditd-acquired-creds-${id}`}
+          field="auditd.session"
+          value={session}
+          iconType="number"
+        />
+      </TokensFlexItem>
+      <TokensFlexItem grow={false}>
+        <PrimarySecondaryUserInfo
+          id={`auditd-was-authorized-${id}`}
+          userName={userName}
+          primary={primary}
+          secondary={secondary}
+        />
+      </TokensFlexItem>
+      {hostName != null && <TokensFlexItem grow={false}>@</TokensFlexItem>}
```

Comment:
I feel like `@` should in the same flex box than hostname with `span`, it is just a decoration to the hostname. and span will be just inline with hostname.

it will fill more semantically correct with the HTML, no need to have another div

Reply:
if you agree with the above it can be apply everywhere else

Reply:
Yes, I do agree and have applied to PR's 2 through 5 now.

Reply:
Thanks for the code for it as well! Really appreciate the help.